.github/workflows/tests.yaml

```diff
@@ -16,13 +16,14 @@ on:
       git-ref:
         description: 'Which git ref to checkout'
         type: string
-        default: 'main'
+        default: ${{ github.ref }}
```

Comment:
`github.ref` is "refs/heads/main"
`github.ref_name` is "main"

[_source_](https://github.com/szepeviktor/byte-level-care/actions/runs/7576269316/job/20634685938)

Reply:
yes, but is there a reason to prefer `main` over `refs/heads/main`?  the latter seems more specific to me